src/.#Yocto 10.hs

```diff
@@ -0,0 +1 @@
+noahjones@Noah-MBP.local.73354
```

Comment:
What are all these small .hs files?

Reply:
These are autosave files created by emacs. Emacs should only keep one autosave file, so it seems to have gone berserk when I ran something in cabal or stack.